Benefits

jsmaks opened the pull request.

jsmaks committed:
добавил секции benefits и problems

jsmaks committed:
Переделал спрайт. и Код Benefits & Problems

Передалал спрайт. В коде написал тире через псведво элемент. Картинку на секции problems burn сделал на плоктности пикселе - 2.0.
паршил block-list готов

AlexCanDoIt commented:
1. Перепроверь размеры иконок внутри рамок в секции benefits, на десктопе.
2. Сломан базовый контейнер в секции problems - исправь.

AlexCanDoIt committed:
Merge branch 'master' into benefits

jsmaks committed:
убрал ошибки с падинго в секциях

Benefits и Problems убрал падинги, что б не перебивали базовые. h2 скрыл через вижуали хидинг. Добавил классы на иконки

jsmaks committed:
Merge branch 'benefits' of https://github.com/AlexCanDoIt/goit-markup-team-project into benefits

AlexCanDoIt committed:
Delete sprite.svg
